Tools/TestWebKitAPI/Tests/WTF/URL.cpp

```diff
@@ -540,39 +541,66 @@ TEST_F(WTF_URL, URLRemoveQueryParameters)
     HashSet<String> keyRemovalSet3 { "key2"_s };
     HashSet<String> keyRemovalSet4 { "key"_s, "key1"_s };
 
-    removeQueryParameters(url1, keyRemovalSet2);
+    auto checkRemovedParameters = [](int lineNumber, const Vector<String>& removedParameters, std::initializer_list<String>&& expected) {
+        Vector<String> expectedParameters { WTFMove(expected) };
+        bool areEqual = removedParameters == expectedParameters;
+        EXPECT_TRUE(areEqual);
+        if (areEqual)
+            return;
+
+        WTFLogAlways("Test failed at %s:%d", __FILE__, lineNumber);
```

Comment:
Did you mean to keep those?

Reply:
Yes, that was my intent (as a debugging aid to help folks debug this test in the future, if needed).